engine/execution/computation/manager.go

```diff
@@ -71,11 +60,10 @@ type ComputationManager interface {
 }
 
 type ComputationConfig struct {
-	CadenceTracing           bool
-	ExtensiveTracing         bool
-	DerivedDataCacheSize     uint
-	ScriptLogThreshold       time.Duration
-	ScriptExecutionTimeLimit time.Duration
+	QueryConfig          query.Config
```

Comment:
if you rename query.Config to query.QueryConfig, you can use anonymous composition / reduce the number of changes